frontend/src/app/home/homepage.component.scss

```diff
@@ -1,5 +1,5 @@
 volontulo-homepage-offer {
```

Comment:
Two things here:
First: this is a selector for the same element as .card so you could delete this rule.
Second: We should be more precise with the width - I'll show you how I would solve it. This is how a .card class would look like in my code:

```
// Bootsrap's default card margin is 15px
$card-margin: 15px;
.card {
  margin: 0 $card-margin 40px $card-margin;
  flex: 1 0 calc(33.333% - #{2*$card-margin});
  max-width: calc(33.333% - #{2*$card-margin});
}
```